tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -0,0 +1,34 @@
+from marionette import Marionette
+from gaiatest.apps.base import Base
+from marionette_driver import By, Wait, expected
+
+class Bugzilla_Lite(Base):
+
+    name = 'Bugzilla Lite'
+    _given_username = (By.CSS_SELECTOR, "#login input[type='email']")
+    _given_password = (By.CSS_SELECTOR, "#login input[type='password']")
+    _button_login = (By.CSS_SELECTOR, "#login input[type='submit']")
+    _app_ready_locator = (By.XPATH, "#content header")
+    _dashboard_navigator_locator = (By.ID, 'dashboardNav')
+    
+    def launch (self):
```

Comment:
Nig: As we don't do the wait anymore, we can remove this function. Python knows that it needs to call Base.launch() if nothing is declared.